Source/WTF/wtf/PlatformHave.h

```diff
@@ -609,7 +609,7 @@
 #define HAVE_GCCONTROLLER_REQUIRING_HAPTICS_SQUARING 1
 #endif
 
-#if PLATFORM(MAC)
+#if PLATFORM(MAC) && PLATFORM(IOS_FAMILY)
```

Comment:
I think this can just be `PLATFORM(COCOA)` now.

Reply:
True, I'll fix it.